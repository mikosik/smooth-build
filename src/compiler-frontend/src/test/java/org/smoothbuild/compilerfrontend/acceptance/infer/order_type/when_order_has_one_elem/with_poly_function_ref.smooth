A myId(A a) = a;
result = [myId];
