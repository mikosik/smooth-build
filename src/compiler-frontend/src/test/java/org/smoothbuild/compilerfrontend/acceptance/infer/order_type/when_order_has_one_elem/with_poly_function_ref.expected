A myId<A>(A a)
  = a<>;
[(A)->A] result<A>
  = [myId<A>];
